ome_zarr/io.py

```diff
@@ -150,8 +164,13 @@ class RemoteZarrLocation(BaseZarrLocation):
     """ Uses the :module:`requests` library for accessing Zarr metadata files. """
```

Comment:
This docstring needs updating?

Reply:
Definitely. Still fighting with Windows.